Makefile

```diff
@@ -34,11 +34,7 @@ JSON_FILES     := $(filter-out $(SOURCE_JSON),$(wildcard $(JSON_FOLDER)/*.json))
 TRF_FILES      := $(subst $(JSON_FOLDER),$(TRF_FOLDER),$(JSON_FILES:.json=.trf))
 
 ifeq ($(OS),Windows_NT)
-	ifeq ($(TERM),cygwin)
```

Comment:
We've already changed this python detection mechanism several times, can you post a source on why this is the correct way to detect it across all platforms?

Reply:
> We've already changed this python detection mechanism several times, can you post a source on why this is the correct way to detect it across all platforms?

py -3 is a good choice for windows CMD and/or powershell. But compare $TERM with cygwin is not a correctly way. Normally "$TERM == cygwin" will always false when using CMD/Powershell, even MSYS terminal. Then we will always go to "PY3: py" branches. That's where the problem lies. If developers install both CPython_Win32 and cpython_mingw, "PY3: py" will use cpython_mingw which will cause "Win32 error 6". So we need to use "py -3" for workaround this.